tests/test-sources/plugins/by-name/avante/default.nix

```diff
@@ -0,0 +1,200 @@
+{
+  empty = {
+    plugins.avante.enable = true;
+  };
+
+  defaults = {
+    plugins.avante = {
+      enable = true;
+
+      settings = {
+        debug = false;
+        provider = "claude";
+        auto_suggestions_provider = "claude";
+        tokenizer = "tiktoken";
+        system_prompt = ''
+          You are an excellent programming expert.
+        '';
+        openai = {
+          endpoint = "https://api.openai.com/v1";
+          model = "gpt-4o";
+          timeout = 30000;
+          temperature = 0;
+          max_tokens = 4096;
+          "__rawKey__'local'" = false;
```

Comment:
Is this rawkey needed? Shouldn't our `isLuaKeyword` logic already handle this case when rendering the lua table?

Reply:
How should I use it ? It's quite new to me ? Simply `local = false;` ?

Reply:
Yeah. That should work. If it doesn't then `toLua` needs some bug fixing 😁